drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/PatternRowBuilder.java

```diff
@@ -60,15 +62,20 @@ public PatternRowBuilder(final Sheet sheet,
         this.patternRow = sheet.createRow(PATTERN_ROW);
     }
 
-    public void build() {
+    public void build(final NotificationReporter notificationReporter) {
 
         final List<BaseColumn> expandedColumns = dtable.getExpandedColumns();
 
         for (; sourceIndex < expandedColumns.size(); sourceIndex++) {
 
             final BaseColumn baseColumn = expandedColumns.get(sourceIndex);
 
-            if (baseColumn instanceof BRLActionVariableColumn) {
+            if (Skipper.shouldSkip(notificationReporter, baseColumn)
+                    || baseColumn instanceof RowNumberCol52
+                    || baseColumn instanceof RuleNameColumn) {
```

Comment:
Why do we need to skip `RowNumberCol52` and `RuleNameColumn` for `PatternRowBuilder` and not for `DataBuilder`. Can't we reimplement `Skipper.shouldSkip` to incorporate also check for `RuleNameColumn` and `PatternRowBuilder`?